counterexamples/transportation/segment/road/restrictions/bad-access-mode.yaml

```diff
@@ -6,20 +6,20 @@ geometry:
   coordinates: [[0, 0], [1, 1]]
 properties:
   # Custom user properties.
-  extBaz: I am a custom user property...
+  ext_baz: I am a custom user property...
   # Overture properties
   theme: transportation
   type: segment
-  updateTime: "2023-02-23T00:02:30-08:00"
+  update_time: "2023-02-23T00:02:30-08:00"
   version: 0
-  subType: road
+  subtype: road
   level: -1
   width: 10
   connectors: [fooConnector, barConnector]
   road:
     class: secondary
     surface: gravel
-    flags: [isLink, isTunnel, isPrivate]
+    flags: [is_link, is_tunnel, isPrivate]
```

Comment:
seems like isPrivate was overlooked

Reply:
Overlooked ... Nice find!

The `isPrivate` flag is deprecated because it doesn't really work as a flag, so I just missed it. I'll fix the counterexample and resubmit.

Reply:
Fixed and updated.